### DISCUSSION TITLE
书面翻译的原文索引不显示

### DISCUSSION THREAD

**Discussion by jht18** [tuna/thuthesis#775] [Q&A] [answered]
使用TexStudio作为编译工具，添加书面翻译后不显示它的原文索引
![图片](https://user-images.githubusercontent.com/104989187/166922308-309c4599-9722-4d03-971c-f4524d25ab80.png)

**Comment by zepinglee** [comment, marked as answer]
参考文档：
<img width="685" alt="Screen Shot 2022-05-05 at 20 32 40" src="https://user-images.githubusercontent.com/12290822/166923706-0b919c38-b259-4113-aa4e-b8d7d220ef01.png">

建议使用 `latexmk` 编译。

**Reply by Sergey3557** [reply]
你好，我在写外文文献的翻译过程中参考文献部分也遇到了类似的问题，请问该如何在texstudio中使用latexmk进行编译呢？

**Reply by zepinglee** [reply]
我对 texstudio 不太了解。

**Reply by zhanglt1027** [reply]
> 你好，我在写外文文献的翻译过程中参考文献部分也遇到了类似的问题，请问该如何在texstudio中使用latexmk进行编译呢？

今天我也遇到了这个问题，可以看看我新发的comment

**Comment by zhanglt1027** [comment]
用了两个小时终于解决了！我也用的是Tex `Studio，外文翻译的参考文献引用处是问号，最后文献列表那里显示的是以前模板里的文献。仔细阅读了说明文档3.3.3这里，意识到是.aux文件没更新，需要手动用命令行（cmd）来更新。把-translation和-index这两条命令手动更新后，参考文献和原文索引就都没问题了。

**Reply by Sergey3557** [reply]
感谢你的回复！我后来也解决啦，我直接在终端/命令行输入latexmk命令可以解决这个问题。调用文件夹后输入latexmk -pdf -xelatex thuthesis-example.tex即可，而且我感觉这个方法程序跑的比texstudio快得多，成功率也高，很不错！
